### PR TITLE
feat(route): add Quicker动作更新

### LINKED ISSUE #9346: Quicker动作今日更新
### 类型

程序更新

### 网站地址

https://getquicker.net/Share/Recent

### 网站描述

Quicker动作今日更新列表

### 需要生成什么内容？

动作名称，ICON，动作描述，动作链接

### 额外描述

_No response_

### 这不是重复的 RSS 请求

- [X] 我已经搜索了[现有 issue](https://github.com/DIYgod/RSSHub/issues) 和 [pull requests](https://github.com/DIYgod/RSSHub/pulls)，以确保该 RSS 尚未被请求。

### PR DESCRIPTION
**Pull request by nczitzk**
## 该 PR 相关 Issue / Involved issue

Close #9346

## 完整路由地址 / Example for the proposed route(s)

```routes
/quicker/versions
/quicker/share/Recent
/quicker/qa/1
/quicker/user/Actions/3-CL
```

## 新 RSS 检查列表 / New RSS Script Checklist
  
- [x] 新的路由 New Route
  - [x] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [x] 文档说明 Documentation
  - [x] 中文文档 CN
  - [ ] 英文文档 EN
- [x] 全文获取 fulltext
  - [x] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [x] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [x] 可以解析 Parsed
  - [x] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

### CODE REVIEW

File: docs/programming.md [hunk 1]
```diff
@@ -501,6 +501,56 @@ GitHub 官方也提供了一些 RSS:
 
 <Route author="hellodword" example="/project-zero-issues" path="/project-zero-issues" />
 
+### 动作分享
```
Comment (TonyRL) [inline docs/programming.md:504]: Missing H2 Header

**Comment by TonyRL** [comment]
It seems `parseRelativeDate()` need some update for this.

**Comment by nczitzk** [comment]
> It seems `parseRelativeDate()` need some update for this.

WIP 😊

**Comment by TonyRL** [comment]
`/quicker/share/Recent` has undefined item

<details>
<summary>Screenshot</summary>

image
</details>

There are still some `Invalid Date` items on:
`/quicker/share/NewActions`
`/quicker/user/Actions/3-CL`

`/quicker/qa` does not work with the new `parseRelativeDate()`

**Pull request merged**